packages/cli-utils/src/middleware/environment.js

```diff
@@ -0,0 +1,187 @@
+const Web3 = require('web3')
```

Comment:
What do you think of using a similar `environment.js` file as: https://github.com/aragon/aragon-cli/blob/91200d3f2f0051b45a344d04b555e242ad666041/packages/aragon-cli/src/middleware/environment.js